packages/venia-ui/lib/components/Accordion/accordion.css

```diff
@@ -0,0 +1,11 @@
+.root {
+    border: 1px solid lightgray;
```

Comment:
There should be an existing CSS variable for our border color that you can use (`--venia-border`).

Reply:
Thanks for catching this, I meant to comment on it.  I was forgetting the `rgb()` in front of `var(---venia-border)` 😨

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2090/commits/ae7b1ae4cf2bdb04f9da1c9632295ffddcd37ad0.